GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -1069,7 +1069,7 @@ + (void)proxySceneDelegateIfNeeded:(UIScene *)scene {
 
   // Skip proxying if the class has a prefix of kGULAppDelegatePrefix, which means it has been
```

Comment:
Maybe change the comment as well.

Reply:
Yeah, it's a good idea. Sorry I forgot about it. I would prefer to do it separately to not disturb Travis.

Reply:
No problem! Thanks for the fix!